daemon/logs.go

```diff
@@ -125,38 +126,53 @@ func (daemon *Daemon) ContainerLogs(name string, config *ContainerLogsConfig) er
 			}
 		}
 	}
+
 	if config.Follow && container.IsRunning() {
-		errors := make(chan error, 2)
-		wg := sync.WaitGroup{}
+		var (
```

Comment:
I hate this block, err already exists. instead of `stdoutPipe` better to use value from config. chans can be just :=